x-pack/legacy/plugins/graph/public/angular/templates/_sidebar.scss

```diff
@@ -27,7 +27,7 @@
 
 .gphSidebar__panel{
   max-height: $euiSizeL * 10;
-  overflow-y: auto;
+  overflow-y: hidden auto;
```

Comment:
Somehow setting `overflow-y` to auto also set `overflow-x` to auto which causes a horizontal scroll bar here because of the way the bootstrap forms are working.